x/treasury/internal/keeper/keeper.go

```diff
@@ -67,34 +67,33 @@ func (k Keeper) Send(ctx sdk.Context, fromDid did.Did, toDidOrAddr string, amoun
 }
 
 func (k Keeper) OracleTransfer(ctx sdk.Context, fromDid did.Did,
-	toDidOrAddr string, oracleDid did.Did, amount sdk.Coins) sdk.Error {
+	toDidOrAddr string, oracleDid did.Did, amount sdk.Coins) error {
 
 	// Check if oracle exists
 	if !k.oraclesKeeper.OracleExists(ctx, oracleDid) {
-		return sdk.ErrInternal("oracle specified is not a registered oracle")
+		return sdkerrors.Wrap(types.ErrInternal, "oracle specified is not a registered oracle")
 	}
 
 	// Confirm that oracle has the required capabilities
 	oracle := k.oraclesKeeper.MustGetOracle(ctx, oracleDid)
 	for _, c := range amount {
 		if !oracle.Capabilities.Includes(c.Denom) {
-			return sdk.ErrInternal(fmt.Sprintf(
-				"oracle does not have capability to send %s", c.Denom))
+			return sdkerrors.Wrap(types.ErrInternal, "oracle does not have capability to send")
```

Comment:
The c.Denom value is important in this message. Please add it back.

Reply:
used wrapf